<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diffs</class>
 <widget class="QDialog" name="Diffs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diffs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/monokuma-green.png</normaloff>:/monokuma-green.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QCheckBox" name="chkNotBoth">
             <property name="text">
              <string>Show Files Not in Both Dirs</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="chkShowSame">
             <property name="font">
              <font>
               <family>Meiryo UI</family>
              </font>
             </property>
             <property name="text">
              <string>Show Identical Files</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeResults">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="indentation">
          <number>15</number>
         </property>
         <column>
          <property name="text">
           <string>Results (0)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lblDir1">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIR 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Translated</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtTranslated1">
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtOriginal1">
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtComments1">
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblDir2">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIR 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Translated</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtTranslated2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Original</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtOriginal2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtComments2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>375</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeResults</tabstop>
  <tabstop>txtTranslated2</tabstop>
  <tabstop>txtOriginal2</tabstop>
  <tabstop>txtComments2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Diffs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Diffs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeResults</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>Diffs</receiver>
   <slot>changedSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowSame</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Diffs</receiver>
   <slot>show_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNotBoth</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Diffs</receiver>
   <slot>show_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>changedSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>doubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>changedSearchFilter()</slot>
  <slot>show_files()</slot>
 </slots>
</ui>
